### ISSUE TITLE
Error after selecting the drop down of saved outfits

### ISSUE THREAD

**Issue by Dizzard92** [WoW-MogIt/MogIt#4]
Received the following after selecting the drop down of saved outfits at the transmogrifier 

Message: Interface\AddOns\MogIt_SetTransmog\Core.lua:180: attempt to index local 'outfit' (a number value)
Time: Fri Jul 22 18:30:05 2022
Count: 2
Stack: Interface\AddOns\MogIt_SetTransmog\Core.lua:180: attempt to index local 'outfit' (a number value)
[string "=[C]"]: ?
[string "@Interface\AddOns\MogIt_SetTransmog\Core.lua"]:180: in function `initFunction'
[string "@Interface\SharedXML\UIDropDownMenu.lua"]:78: in function `UIDropDownMenu_Initialize'
[string "@Interface\SharedXML\UIDropDownMenu.lua"]:1075: in function <Interface\SharedXML\UIDropDownMenu.lua:933>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `ToggleDropDownMenu'
[string "@Interface\AddOns\MogIt\Libs\Libra\Dropdown.lua"]:79: in function `Toggle'
[string "@Interface\AddOns\MogIt_SetTransmog\Core.lua"]:244: in function <Interface\AddOns\MogIt_SetTransmog\Core.lua:243>

Locals:

**Comment by Dizzard92** [comment]
Version 9.2.5

**Comment by Wyr3d** [comment]
Yeah, it's wardrobe outfits - I'm assuming the link to the in-game transmog is broken.
